src/renderer/src/components/account-tooltip.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component, PropTypes } from 'react';
 import { Button } from 're-bulma';
 import B from '../lib/bem';
+import T from '../lib/twitter-client';
```

Comment:
oops.., sorry, I had merged..
 Please remove this line, if you will request next PR.
we have not to call twitter client API from component.
we should call api from `sagas`.